### ISSUE TITLE
manifoldapp.org website update: accessibility

### ISSUE THREAD

**Issue by mkgold** [ManifoldScholar/manifold#2411]
From Slack: "Do we have a page on the Manifold marketing website where we talk about accessibility? Especially given what the Washington group has told us about how we compare w other platforms they evaluated, I think we should foreground our work to make Manifold accessible (and also note that we approach it as a process). That kind of page would also allow us to credit the UW group."

**Comment by zdavis** [comment]
@mkgold @susanmpls @tsmyre We need a better place to track things like this. Having it here is a problem because it's not really an actionable task for the manifold platform itself.

Any thoughts on where we should track this? We have a private github repository for the marketing website (because it's not open source). We track issues there internally, but if we add everyone to that it will cost us $100+/month.

We could use Basecamp to track this kind of task, if that makes sense to y'all. We haven't used it for a while, but it does have todo lists that we could use for this.

**Comment by tsmyre** [comment]
What about Trello? Their free plan looks like it might work for this, no?

https://trello.com/pricing

**Comment by zdavis** [comment]
Or we could use a project board in this repo so I don't have to sign into Trello? I don't mind having a project for it (and making it public), I just don't want to put this stuff in issues.

**Comment by tsmyre** [comment]
Yeah that works for me!

**Comment by mkgold** [comment]
that works for me as well
